cmd/cli/plugin/secret/registry_secret_add.go

```diff
@@ -54,7 +54,7 @@ func init() {
 	registrySecretAddCmd.MarkFlagRequired("username") //nolint
 }
 
-func registrySecretAdd(cmd *cobra.Command, args []string) error {
+func registrySecretAdd(cmd *cobra.Command, args []string) error { //nolint:gocyclo
```

Comment:
If we are adding `nolint`, could we make it specific? For example `// nolint:gocyclo,funlen`

Reply:
it is only gocyclo 
no issue with funlen

Reply:
^ could you address

Reply:
Fixed by breaking into functions